CMP1903M Assessment 1 Base Code/Program.cs

```diff
@@ -16,19 +16,19 @@ static void Main()
 
             //Create 'Input' object
             //Get either manually entered text, or text from a file
-
-
+            var input = new Input();
+            input.inputChoice();
+            
             //Create an 'Analyse' object
             //Pass the text input to the 'analyseText' method
-
-
-            //Receive a list of integers back
-
+            var analyse = new Analyse();
+            analyse.analyseText(input.text);
 
             //Report the results of the analysis
+            var report = new Report();
+            report.outputReport(analyse.values);
 
-
-            //TO ADD: Get the frequency of individual letters?

```

Comment:
If possible add frequency of letters if more time was available as it may boost a grade but not completely mandatory and still very usable without it